src/NServiceBus.RabbitMQ.Tests/RabbitMqContext.cs

```diff
@@ -99,30 +69,39 @@ public void TearDown()
             channelProvider?.Dispose();
         }
 
-        protected IncomingMessage WaitForMessage()
+        protected bool TryWaitForMessageReceipt()
```

Comment:
The way I looked at it was: a method which waited for message receipt and timeout, throwing a `TimeOutException`, would be called `WaitForMessageReceipt`, and the method which tries to do that, and returns a `bool` to indicate whether the attempt succeeded would then be `TryWaitForMessageReceipt`. This follows the pattern of methods like `Get` and `TryGet`.